src/main/kotlin/no/nav/tilleggsstonader/kontrakter/søknad/SøknadsskjemaBarnetilsyn.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.tilleggsstonader.kontrakter.søknad
+
+import no.nav.tilleggsstonader.kontrakter.søknad.barnetilsyn.AktivitetAvsnitt
+import no.nav.tilleggsstonader.kontrakter.søknad.barnetilsyn.BarnAvsnitt
+import no.nav.tilleggsstonader.kontrakter.søknad.barnetilsyn.HovedytelseAvsnitt
+
+data class SøknadsskjemaBarnetilsyn(
```

Comment:
Ikke direkte koblet til denne PRen, men hva bruker vi `Søknadsskjema` til og hvorfor er ikke `SøknadsskjemaBarnetilsyn` et sånt skjema? 🤓 

Nå som det er lenge siden jeg har sett på dette var det i hvert fall litt utydelig for meg hva de forskjellige klassene skal brukes til.

Reply:
`Søknadsskjema` inneholder obligatoriske felter for en søknad, ident, type, språk
`SøknadsskjemaBarnetilsyn` er selve søknaden

Navnene er forvirrende, og burde vel egentlige endre de til noe annet smart